swift/ql/lib/codeql/swift/frameworks/StandardLibrary/CustomUrlSchemes.qll

```diff
@@ -14,10 +15,15 @@ private class CustomUrlRemoteFlowSource extends SourceModelCsv {
         ";UIApplicationDelegate;true;application(_:open:options:);;;Parameter[1];remote",
         ";UIApplicationDelegate;true;application(_:handleOpen:);;;Parameter[1];remote",
         ";UIApplicationDelegate;true;application(_:open:sourceApplication:annotation:);;;Parameter[1];remote",
-        // TODO: The actual source is the value of `UIApplication.LaunchOptionsKey.url` in the launchOptions dictionary.
+        // TODO 1: The actual source is the value of `UIApplication.LaunchOptionsKey.url` in the launchOptions dictionary.
         //       Use dictionary value contents when available.
         // ";UIApplicationDelegate;true;application(_:didFinishLaunchingWithOptions:);;;Parameter[1].MapValue;remote",
         // ";UIApplicationDelegate;true;application(_:willFinishLaunchingWithOptions:);;;Parameter[1].MapValue;remote"
+        // TODO 2: MaD doesn't seem to take into account extensions adopting to protocols even if the subtypes column is set to true.
+        // ";UIWindowSceneDelegate;true;scene(_:continue:);;;Parameter[1];remote",
```

Comment:
I hope https://github.com/github/codeql/pull/11935 fixes this todo. It's waiting on @geoffw0 to resolve a merge conflict, and then I think it's good to go.

Reply:
Great! Happy to get rid of the TODO and the workaround before merging this once #11935 has landed.

Reply:
I confirm it now works properly when using MaD rows instead of plain CodeQL to model those sources. Thanks!

Reply:
#11935 is now merged.

Reply:
Rebased and got rid of the TODO! 😄